vcbuild.bat

```diff
@@ -543,16 +548,19 @@ echo Failed to create vc project files.
 goto exit
 
 :help
-echo vcbuild.bat [debug/release] [msi] [test/test-ci/test-all/test-addons/test-addons-napi/test-internet/test-pummel/test-simple/test-message/test-gc/test-tick-processor/test-known-issues/test-node-inspect/test-check-deopts/test-npm/test-async-hooks/test-v8/test-v8-intl/test-v8-benchmarks/test-v8-all] [ignore-flaky] [static/dll] [noprojgen] [small-icu/full-icu/without-intl] [nobuild] [nosnapshot] [noetw] [noperfctr] [licensetf] [sign] [ia32/x86/x64] [vs2015/vs2017] [download-all] [enable-vtune] [lint/lint-ci/lint-js/lint-js-ci] [package] [build-release] [upload] [no-NODE-OPTIONS] [link-module path-to-module] [debug-http2] [debug-nghttp2] [clean]
+echo vcbuild.bat [debug/release] [msi] [test/test-ci/test-all/test-addons/test-addons-napi/test-internet/test-pummel/test-simple/test-message/test-gc/test-tick-processor/test-known-issues/test-node-inspect/test-check-deopts/test-npm/test-async-hooks/test-v8/test-v8-intl/test-v8-benchmarks/test-v8-all] [ignore-flaky] [static/dll] [noprojgen] [small-icu/full-icu/without-intl] [nobuild] [nosnapshot] [noetw] [noperfctr] [licensetf] [sign] [ia32/x86/x64] [vs2015/vs2017] [download-all] [enable-vtune] [lint/lint-ci/lint-js/lint-js-ci] [package] [build-release] [upload] [no-NODE-OPTIONS] [link-module path-to-module] [debug-http2] [debug-nghttp2] [openssl-system-ca-path path-to-system-certs] [clean]
 echo Examples:
-echo   vcbuild.bat                          : builds release build
-echo   vcbuild.bat debug                    : builds debug build
-echo   vcbuild.bat release msi              : builds release build and MSI installer package
-echo   vcbuild.bat test                     : builds debug build and runs tests
-echo   vcbuild.bat build-release            : builds the release distribution as used by nodejs.org
-echo   vcbuild.bat enable-vtune             : builds nodejs with Intel VTune profiling support to profile JavaScript
-echo   vcbuild.bat link-module my_module.js : bundles my_module as built-in module
-echo   vcbuild.bat lint                     : runs the C++ and JavaScript linter
+echo   vcbuild.bat                             : builds release build
+echo   vcbuild.bat debug                       : builds debug build
+echo   vcbuild.bat release msi                 : builds release build and MSI installer package
+echo   vcbuild.bat test                        : builds debug build and runs tests
+echo   vcbuild.bat build-release               : builds the release distribution as used by nodejs.org
+echo   vcbuild.bat enable-vtune                : builds nodejs with Intel VTune profiling support to profile JavaScript
+echo   vcbuild.bat link-module my_module.js    : bundles my_module as built-in module
+echo   vcbuild.bat lint                        : runs the C++ and JavaScript linter
+echo   vcbuild.bat openssl-system-ca-path path : builds using the specified path ^
+to system CA (PEM format) in addition to the OpenSSL supplied CA store or ^
+compiled-in Mozilla CA copy
```

Comment:
It's just the Mozilla CA store.  OpenSSL doesn't provide certs, it only validates them. =)

Reply:
Oh right, I'll update the description. Thanks!